spryker/usr/local/share/spryker/spryker_functions.sh

```diff
@@ -49,10 +49,16 @@ do_spryker_app_permissions() {
 
 do_spryker_config_create() (
   set +x
-  echo "Creating Postgres Credentials file in /root/.pgpass"
+  if [ "$NON_PRIVELEGED_USER" = "true" ]; then
+    local target=~/.pgpass
```

Comment:
I suppose technically you can use ~/ for root too

Reply:
You're right. Should I use `~` for all cases?

Reply:
it wouldn't hurt to